test/e2e/specs/help-center/help-center__general-interaction.ts

```diff
@@ -2,11 +2,141 @@
+describe.each( [
+	{ accountName: 'defaultUser' as TestAccountName },
+	{ accountName: 'atomicUser' as TestAccountName },
+] )( 'Help Center: Verify Help Center is accessible', function ( { accountName } ) {
+	let page: Page;
+	let supportComponent: SupportComponent;
+	let testAccount: TestAccount;
+	const helpCenterContainerVisibilityErrorMessage = `This is a bug that should be urgently fixed.
+	But because this test runs against ETK production, this bug was probably not introduced in this pull request.
+	Please consider alerting the last person who deployed ETK to attend to this issue and fix the Help Center.`;
+
+	beforeAll( async function () {
+		page = await browser.newPage();
+
+		testAccount = new TestAccount( accountName );
+		await testAccount.authenticate( page, { waitUntilStable: true } );
+
+		supportComponent = new SupportComponent( page );
+	} );
+
+	afterAll( async function () {
+		await page.close();
+	} );
+
+	describe( 'Verify Help Center is opened and visible in Calypso', function () {
+		it( 'Verify Help Center is initially closed', async function () {
+			expect( await page.locator( '.help-center__container' ).isVisible() ).toBeFalsy();
+		} );
+
+		it( 'Open Help Center', async function () {
+			await supportComponent.openPopover();
+		} );
+
+		it( 'Verify Help Center is opened', async function () {
+			expect( await page.locator( '.help-center__container' ).isVisible() ).toBeTruthy();
+		} );
+	} );
+
+	describe( 'Verify Help Center is opened and visible in Editor', function () {
+		it( 'Navigate to the Editor and verify the Help Center is initially closed', async function () {
+			const postURL = DataHelper.getCalypsoURL(
+				'/post/' + testAccount.getSiteURL( { protocol: false } )
+			);
+
+			await page.goto( postURL, {
+				waitUntil: 'networkidle',
+			} );
+		} );
+
+		it( 'Verify Help Center is initially closed', async function () {
+			expect( await page.locator( '.help-center__container' ).isVisible() ).toBeFalsy();
+		} );
+
+		it( 'Open Help Center', async function () {
+			// For Help Center loaded within iframe
+			const helpCenterButtonIframe = page
+				.frameLocator( '.calypsoify iframe' )
+				.getByLabel( 'Help', { exact: true } );
+
+			const helpCenterButtonWithoutIframe = page.locator( 'button.help-center' );
+
+			const helpCenterButton = ( await helpCenterButtonIframe.isVisible() )
+				? helpCenterButtonIframe
+				: helpCenterButtonWithoutIframe;
+
+			await helpCenterButton.click();
+		} );
+
+		it( 'Verify Help Center is opened', async function () {
+			const helpCenterContainerIframe = page
+				.frameLocator( '.calypsoify iframe' )
+				.locator( '.help-center__container' );
+
+			const helpCenterContainerWithoutIframe = page.locator( '.help-center__container' );
+
+			const helpCenterContainer = ( await helpCenterContainerIframe.isVisible() )
+				? helpCenterContainerIframe
+				: helpCenterContainerWithoutIframe;
```

Comment:
+1. Let's avoid interacting with anything outside the iframe. The iframe takes 100% of the screen and anything outside it is unreachable.